README.md

```diff
@@ -4,41 +4,29 @@
 
 [Get it on the VS Code Marketplace!](https://marketplace.visualstudio.com/items?itemName=ms-azure-devops.azure-pipelines)
 
-This VS Code extension brings syntax highlighting and autocompletion for
-Azure Pipelines YAML to VS Code. Basic YAML validation is built in to VS Code, but
-now you can have syntax highlighting that's aware of the Pipelines YAML schema.
-This means that you get red squigglies if you say `tasks:` where you meant `task:`.
-IntelliSense is also schema-aware. Wherever you are in the file, press Ctrl-Space
-(Cmd-Space on macOS) to see what options you have at that point.
-
-By default, the extension will highlight known Azure Pipelines files in the root
-of your workspace. You can change the language mode at the lower right to work
-with one file at a time. Click the language picker, then choose "Azure Pipelines".
-If you have files which should always use this extension, set your user or
-workspace settings to match those file paths with this extension. For example:
-
-```yaml
+This VS Code extension helps to seamlessly create, build and deploy in continuous manner using Azure Pipelines to **Azure WebApp Service** without leaving the **Visual Studio Code interface**. Also, this extension brings syntax highlighting that's aware of the Pipelines YAML schema and autocompletion for Azure Pipelines YAML to VS Code.
+
+For setting up Azure Pipelines using this extension you can invoke Azure Pipelines < Configure Pipeline from command palette (Ctrl +  Shift + P) or Right-Click in File explorer.  The guided workflow will auto generate a modifiable YAML file defining the build and deploy process which will be added to your repository for future reference.
```

Comment:
Commas missing use below: 
The guided workflow will auto generate a modifiable YAML file, defining the build and deploy process, which will be added to your repository for future reference.